charts/ceph-csi-rbd/values.yaml

```diff
@@ -107,6 +107,15 @@ provisioner:
   replicaCount: 3
   # Timeout for waiting for creation or deletion of a volume
   timeout: 60s
+  # Hard limit for maximum number of nested volume clones that are taken before
+  # a flatten occurs
+  hardMaxCloneDepth: 8
+  # Soft limit for maximum number of nested volume clones that are taken before
+  # a flatten occurs
+  softMaxCloneDepth: 4
+  # Flatten all cloned rbd images upon creation, if kernel version doesnot
+  # support deep-flatten feature
+  flattenOnCreate: true
```

Comment:
may `enabledeepflatten` would be a better naming I believe.

Reply:
The name was suggested by @dillaman in one of the discussion, This seems correct as we need to flatten the image upon creation, I don't have hard feelings I can change it if required.

Reply:
No, thats fine. lets keep it as `flattenOnCreate`

Reply:
Alternative suggestion could be `enableThinProvisionedClones` (or similar). I don't think anything referencing the RBD `deep-flatten` feature bit would mean much to end-users.

Reply:
May be what we do internally - `flatten at create` or `do it in a different queue` also may not add value to the user. in that sense something like `enableThinProvisionedClone` looks better to me.